docs/src/pages/guides/notifications.md

```diff
@@ -1,3 +1,189 @@
-# Notifications
+---
+id: notifications
+title: Notifications
+---
 
-under construction
+## Notification Types
+
+In Monika, currently we provided multiple notifications type. We are implementing email notifications through sendgrid, mailgun, smtp and webhook.
+
+## Configurations
+
+Notifications can be send through multiple processes and mail accounts, define through configurations as in the config.json.example.
```

Comment:
You can configure notifications as much as you need with different or same types. To configure notifications you need to define it on `config.json` in `notifications` section.

Is it better?

Reply:
modified